### DISCUSSION TITLE
Is there a DLT API to get a list of all AppId and CtxtId, and their log levels

### DISCUSSION THREAD

**Discussion by 8vp** [COVESA/dlt-daemon#684] [Q&A] [answered]
Hi all, we are looking for a DLT Daemon API that we can call to fetch all AppIds, associated CtxtIds and log levels for the pairs. We can see dlt_get_appid and dlt_env_II_item, but they dont seem to give all these details. If anyone can please guide us?

**Comment by lti9hc** [comment, marked as answer]
Hello @8vp,
we have dlt_daemon_find_users_list API, it will provide all application
I think it is useful for you

**Reply by 8vp** [reply]
Thank you !
